cuda_core/tests/conftest.py

```diff
@@ -21,22 +21,33 @@
 from cuda.core.experimental._utils import handle_return
 
 
+@pytest.fixture(scope="session", autouse=True)
+def always_init_cuda():
+    handle_return(driver.cuInit(0))
```

Comment:
FYI, @ksimpson-work the CI was able to catch this issue: Depending on how the tests are run, it could be possible that a test ends without CUDA even initialized. So we must ensure it ourselves by the test start time.